src/core_modules/capture-core/components/WidgetProfile/OverflowMenu/OverflowMenu.component.js

```diff
@@ -45,13 +50,15 @@ export const OverflowMenuComponent = ({
                                 <MenuDivider dense />
```

Comment:
What do you think about moving `MenuDivider` below along with `DeleteMenuItem`? It looks a bit odd right now when only the changelog is enabled.
<img width="292" alt="Screenshot 2025-01-16 at 08 43 32" src="https://github.com/user-attachments/assets/5c75c09e-2c0b-4afa-9a1e-5ea3e09258ca" />

 We can also completely remove the MenuDivider as it is in the StagesAndEvents widget, which IMHO looks cleaner.
<img width="736" alt="Screenshot 2025-01-16 at 08 49 32" src="https://github.com/user-attachments/assets/acb41ff2-90de-4733-8012-48875b3344e4" />

Thanks!

Reply:
Good catch, @simonadomnisoru! I just removed it, I think that looks pretty clean.